bootstrap-salt.ps1

```diff
@@ -85,7 +85,7 @@ Param(
     [Parameter(Mandatory=$false,ValueFromPipeline=$true)]
     # Doesn't support Python versions prior to "2017.7.0"
     [ValidateSet("2","3")]
-    [string]$pythonVersion = "2",
+    [string]$pythonVersion = "3",
```

Comment:
@twangboy do we need a less interruptive way of changing this? or is this fine?

Reply:
That's fine. It's just changing the default.